pkg/util/remotecommand/constants.go

```diff
@@ -0,0 +1,53 @@
+/*
```

Comment:
pkg/util is an odd location for this. It's actually part of the API. As we don't have a better location for this, moving it to client-go would be fine IMO.